src/script/components/MessagesList/Message/FailedToAddUsersMessage.tsx

```diff
@@ -49,6 +55,7 @@ const config = Config.getConfig();
 const FailedToAddUsersMessage: React.FC<FailedToAddUsersMessageProps> = ({
   isMessageFocused,
   message,
+  errorMessageType = ErrorMessageType.offlineBackEnd,
```

Comment:
The different error messages are not part of the scope of the September release.
The only error message displayed stays the same as offline back-end